packages/webdriver/tests/utils.test.js

```diff
@@ -290,14 +290,33 @@ describe('utils', () => {
     describe('overwriteElementCommands', () => {
         it('should overwrite command', function () {
             const context = {}
+            const origFnMock = jest.fn(() => 1)
             const propertiesObject = {
-                foo: { value() { return 1 } },
+                foo: { value: origFnMock },
                 __elementOverrides__: {
                     value: { foo(origCmd, arg) { return [origCmd(), arg] } }
                 }
             }
             overwriteElementCommands.call(context, propertiesObject)
             expect(propertiesObject.foo.value(5)).toEqual([1, 5])
+            expect(origFnMock.mock.calls.length).toBe(1)
+            expect(origFnMock.mock.instances[0]).toBe(propertiesObject.foo)
+        })
+
+        it('should support rebinding when invoking original fn', function () {
```

Comment:
Can you modify the test so that it returns a different value as expected because it is bound to a different instance?

Reply:
You mean in addition to the check of jest.mock.instances which already tracks this-bindings on invocations? Or do you want replace the checks on the jest-mock completely with checks against the return value?

Reply:
I haven't used `jest.fn().mock.instances` before so it wasn't clear for me that it theoretical also checks that. Might adding another smoke test for this here: https://github.com/webdriverio/webdriverio/blob/master/tests/mocha/test.js#L128 ?

Reply:
Sure, I'll do that.

Reply:
I have added a smoketest.